packages/core/test/EscrowFactory.ts

```diff
@@ -75,75 +75,66 @@ describe('EscrowFactory', function () {
     });
   });
 
-  it('Operator should not be able to create an escrow without staking', async () => {
-    await expect(
-      escrowFactory
-        .connect(operator)
-        .createEscrow([ethers.constants.AddressZero])
-    ).to.be.revertedWith('Needs to stake HMT tokens to create an escrow.');
-  });
-
-  it('Operator should be able to create an escrow after staking', async () => {
-    const event = await stakeAndCreateEscrow(staking);
```

Comment:
why is this test going away?

Reply:
This is a mistake I made, but in the last commit I had added it back again